.github/issue_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Thanks for creating an issue for Cornet.JS.
+     Please fill out this form so we can help you as quickly as possible. -->
```

Comment:
Form? Template?

Reply:
Please do all stuff on GITLAB from now on, please!

Reply:
Okay, see you there